cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java

```diff
@@ -581,12 +581,21 @@ public void upsertTransaction (LinkedHashMap<String, ArrayList<JsonElement>> agg
             String insertQuery = SQLQueryUtils.sqlInsertQuery(aggregation,
```

Comment:
CNR entry (with reference to issue number, if we have some) should be included.

Reply:
Fixed in [2f9140f](https://github.com/telefonicaid/fiware-cygnus/pull/1960/commits/2f9140fe589e08bee19e85ea1daeca198e1e0f97)